cairo_programs/memset.cairo

```diff
@@ -0,0 +1,19 @@
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.memset import memset
+
+func main():
+    alloc_locals
+    let (local strings : felt*) = alloc()
+    memset(strings,'Lambda',20)
+    assert strings[0] = 'Lambda'
+    assert strings[19] = 'Lambda'
```

Comment:
We can use compare arrays to make sure it's true for all elements.

Reply:
(An analog really, but same essence)

Reply:
[Done!](https://github.com/lambdaclass/cleopatra_cairo/pull/303/commits/6e106e036ff92c6cd09ef6e561fc7950f9ae8e4c)